### PR TITLE
Update to 3.16.1

### PR DESCRIPTION
**Pull request by brianjmurrell**
Update packaging/.

### CODE REVIEW

File: fuse-install-fix.patch
Comment (ashleypittman) [inline fuse-install-fix.patch:1]: Do we still need this?

**Review by ashleypittman — DISMISSED** [review]
Test result showing something that previously was failing now working at https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13927/28/testReport/FTEST_dfuse/DFuseReadTest/

File: fuse-install-fix.patch
Comment (brianjmurrell) [reply fuse-install-fix.patch:1]: Yes, I think we do.  It still applies to the upstream source and we are still trying to build a fuse3-compatible RPM.

**Review by ashleypittman — APPROVED** [review]

**Review by JohnMalmberg — APPROVED** [review]

**Comment by ryon-jensen** [comment]
Failures in the daos builds are known.

**Pull request merged**